src/main/java/seedu/address/logic/parser/MarkAttendanceParser.java

```diff
@@ -21,10 +22,11 @@ public class MarkAttendanceParser implements Parser<MarkAttendanceCommand> {
      */
     public MarkAttendanceCommand parse(String args) throws ParseException {
         requireNonNull(args);
-        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_TUTORIAL);
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_TUTORIAL, PREFIX_PARTICIPATION_STATUS);
 
         Index index;
         int tutorial = 0;
+        String status = null;
```

Comment:
Might be safer to create a status class so that you can use the properties of an empty status :))